apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/RetryOptions.cs

```diff
@@ -113,15 +113,15 @@ public static RetryOptions NoRetry(ExceptionHandling exceptionHandling = Excepti
         /// <param name="bufferOverflow">Optional, how a buffer overflow should be handled.
         ///     Defaults to <see cref="BufferOverflow.IgnoreNewEntries"/>.</param>
         /// <param name="bufferSizeBytes">Optional, the size of the buffer to be used in bytes.
-        ///     Defaults to 2^18 = 262144.</param>
-        /// <param name="retryInterval">Optional, the time between retries. Default to 2 seconds.</param>
-        /// <param name="retryAttempts">Optional, the number of retry attempts. Defaults to 10.</param>
+        ///     Defaults to <see cref="DefaultBufferSize"/>.</param>
+        /// <param name="retryInterval">Optional, the time between retries. Default to <see cref="DefaultRetryInterval"/>.</param>
+        /// <param name="retryAttempts">Optional, the number of retry attempts. Defaults to <see cref="DefaultRetryAttempts"/>.</param>
         public static RetryOptions Retry(
             ExceptionHandling exceptionHandling = ExceptionHandling.Ignore,
             BufferOverflow bufferOverflow = BufferOverflow.IgnoreNewEntries,
-            int bufferSizeBytes = DefaultBufferSize,
-            TimeSpan? retryInterval = null, int retryAttempts = DefaultRetryAttempts)
+            int? bufferSizeBytes = null, TimeSpan? retryInterval = null, int? retryAttempts = null)
                 => new RetryOptions(RetryType.Retry, exceptionHandling, bufferOverflow,
-                    bufferSizeBytes, retryAttempts, retryInterval ?? DefaultRetryInterval);
+                    bufferSizeBytes ?? DefaultBufferSize, retryAttempts ?? DefaultRetryAttempts,
```

Comment:
Can we reorder either this method or the RetryOptions constructor to keep the parameter order consistent?